app/src/main/res/values-hu/strings.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!--Generated by crowdin.com-->
+
 <!--
     Copyright (C) 2014-2017 Arpit Khurana <arpitkh96@gmail.com>, Vishal Nehra <vishalmeham2@gmail.com>
 
```

Comment:
Please change the year 2017 to 2018 to reflect changes to this file.

Optional: feel free to add your name+email here as a symbol of your contribution, if you'd like.